src/components/style/navbar.css

```diff
@@ -0,0 +1,501 @@
+.FDSnavfeaturedItemImageContainer {
+  box-shadow: -0.625em 0.625em 0 0 rgb(0 0 0 / 50%);
+  width: 300px;
+  height: 181px;
+}
+
+.FDSnavfeaturedItemImage {
+  display: block;
+}
+
+.FDSnavfeaturedItemText {
+  margin-top: var(--space-l);
+  font-size: 1.25em;
+  color: var(--color-white);
+  font-weight: 900;
+}
+
+.FDSnavfeaturedItemLearnMore {
+  margin-top: var(--space-l);
+  color: var(--color-white);
+  font-size: 1em;
+  font-weight: 700;
+}
+
+.FDSnavfeaturedItemLearnMore:hover {
+  text-decoration: underline;
+}
+
+.FDSnavfeaturedItem {
+  cursor: auto;
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: 0 !important;
+  padding: 35px 35px 0 0 !important;
+  background-color: transparent !important;
+}
+
+.FDSnaviconButton {
+  display: block;
+}
+
+.FDSnavmobileIconButton {
+  display: none;
+}
+
+.FDSnavtopContainer {
+  display: flex;
+}
+
+.FDSnavcontainer {
+  display: flex;
+  position: relative;
+  align-items: center;
+  max-width: 1300px;
+  margin: auto;
+}
+
+.FDSnavhasSubMenu {
+  width: 900px;
+  height: 350px;
+  right: calc(100% - 40em) !important;
+}
+
+.FDSnavdropdownContainer {
+  opacity: 0;
+  visibility: hidden;
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  right: -5em;
+  top: 100%;
+  z-index: calc(var(--zindex-navigation) + 1);
+}
+
+.FDSnavmenuContainer > li > a {
+  padding-right: 30px;
+  padding-left: 30px;
+  width: 263px;
+}
+
+.FDSnavmenuContainer {
+  background-color: #a02385;
+  padding: var(--space-l) 0;
+  width: 100%;
+}
+
+.FDSnavsubMenuList {
+  padding: 0;
+  width: 100%;
+  background-color: #6a1e6d;
+}
+
+.FDSnavsubMenuContainer {
+  padding: var(--space-l) 0;
+  display: none;
+  position: absolute;
+  top: 0;
+  left: 263px;
+  width: calc(100% - 263px);
+  height: 100%;
+  background-color: #6a1e6d;
+}
+
+.FDSnavdropdownItem {
+  padding-left: 20px;
+  padding-right: 20px;
+  display: flex;
+  align-items: center;
+  min-height: 50px;
+  width: 263px;
```

Comment:
should the width extend so that you can get the hover action without needing to hover over the text?